server/db.ts

```diff
@@ -9,6 +9,7 @@ export const initDB = createConnection({
   host: process.env.DB_URL || 'db',
   port: 5432,
   entities: [__dirname + '/models/*.ts'],
+  migrations: [__dirname + '/migrations/**/*.ts'],
```

Comment:
Use await getConfig or something like that I dont know the specific api

Reply:
I'm unsure I understand. Are you referring to adding https://www.npmjs.com/package/getconfig?

Reply:
No, sorry, I was thinking about importing the config, not sure if typeorm can do it itself, but just importing the  typeorm config file and spreading it here?

Reply:
Ah, understood.

If that's the case, we should switch to an `ormconfig.js` (rather than `.json`) so we can support environment variables.

Additionally, it looks like `createConnection` and `ormconig.json` have two different ports (because of the `54320:5432` mapping in `docker-compose`). Unsure how that should be handled. Would require some `if insideOfDockerContainer` type of logic.

We should probably:
a) Change the `docker-compose` mapping to the norm (`5432:5432`)
b) Convert `ormconfig.json` to `ormconfig.js` to support environment variables
c) Finally get rid of the custom config in this file and default to `ormconfig.js`.

Thoughts?

Reply:
@Zeko369 OK, so I was mistaken. I see what you were saying and I think the above implementation satisfies that.

Reply:
I wouldn't change the port mapping because if someone has `psql` running on the system it's on the 5432, so this would make it possible for people to have both

![image](https://user-images.githubusercontent.com/3064377/70866284-0b9b2c00-1f68-11ea-99a3-54ea31cc552f.png)

Reply:
We'll talk about this in the meeting in a hour

Reply:
OK, that's cool. I haven't done that so this PR should be safe.